packages/itwin/geo-tools/package.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "@itwin/geo-tools-react",
+  "version": "1.0.0",
```

Comment:
Are we sure its stable enough to be 1.0?

Reply:
I think so, its been used in production for more than 1 year/

Reply:
@aruniverse

Reply:
I know it's been in Prod in DR for over a yr. I guess I was more worried about it breaking or there being regressions since we're updating major ver of core.

Reply:
I don't really mind about the version number, what is your suggestion?

Reply:
I was thinking 0.x until DR hits Prod with 3.0

Reply:
going to ver as 0.1.0, then will have separate pr to add ui provider and removal of svg?sprite